pages/archiving/setup.rst

```diff
@@ -56,6 +56,127 @@ The archived indices will be stored in a backend. A backend that stores the data
+
++-----------------------------+-------------------------------------------+
+| Name                        | Description                               | 
++=============================+===========================================+
+| Title                       | A simple title to identify the backend    |     
++-----------------------------+-------------------------------------------+
+| Description                 | Longer description for the backend        |     
++-----------------------------+-------------------------------------------+                                                
+| S3 Endpoint URL             | Only configure this if not using AWS      |                                                
++-----------------------------+-------------------------------------------+
+| AWS Authentication Type     | Choose access type from the dropdown menu |     
++-----------------------------+-------------------------------------------+                                                 
+| AWS Assume Role (ARN)       | This is an optional input for             |
+|                             | alternate authentication mechanisms.      |      
++-----------------------------+-------------------------------------------+     
+| Bucket Name                 | The name of the S3 bucket                 |     
++-----------------------------+-------------------------------------------+     
+| Spool Directory             | Temporary name for the archiving process  |    
++-----------------------------+-------------------------------------------+     
+| AWS Region                  | Choose *Automatic* or configure the       |
+|                             | appropriate option                        |
++-----------------------------+-------------------------------------------+
+| S3 Output Base Path         | Creates archive structure                 |
++-----------------------------+-------------------------------------------+
+
+AWS Authentication Type
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Graylog provides several options for granting access. You can:
+
+* use the *Automatic* authentication mechanism if you provide AWS credentials through your file
+  system or process environment.
+* enter credentials manually
+
+AWS Assume Role (ARN)
+~~~~~~~~~~~~~~~~~~~~~
+
+This is typically used for buckets that you don’t own. For example, if someone gave you temporary 
+permissions to their bucket, you would need to use this configuration.
+
+Spool Directory
+~~~~~~~~~~~~~~~
+
+The Spool Directory is always writable for the upload process to persist.
+
+Before Graylog upload archives to the S3, it has to write them into the file system. As soon as 
+Graylog writes these segments, they are uploaded. 
+
+Keep in mind you need 5 gigabytes of free disk space in this directory to manage 10 segments at 
+500 megabytes. However, it may use less memory because it deletes files as soon as they upload.
```

Comment:
```suggestion
The archiving process needs this directory to store some temporary data, before it can be uploaded to S3.

This directory should be writable and have enough space to fit 10 times the *Max Segment Size*  (add link to the config section)
```